### PR TITLE
PIM-4777: Fixed mass edit of an attribute inclued in a variant group

### PR DESCRIPTION
**Pull request by ClementGautier**
| Q                  | A
| ------------------ | ---
| Migration script?  | N/A
| Behats             | Y
| Specs & Static     | Y
| Changelog updated  | N/A

### CODE REVIEW

File: spec/Pim/Bundle/EnrichBundle/Connector/Processor/MassEdit/Product/EditCommonAttributesProcessorSpec.php [hunk 1]
```diff
@@ -48,10 +47,9 @@ function it_sets_the_step_execution(StepExecution $stepExecution)
         $this->setStepExecution($stepExecution)->shouldReturn($this);
     }
 
-    function it_sets_values_to_attributes(
+    function it_dont_sets_values_from_variant_group(
```
Comment (jjanvier) [inline spec/Pim/Bundle/EnrichBundle/Connector/Processor/MassEdit/Product/EditCommonAttributesProcessorSpec.php]: doesnt_set

File: src/Pim/Bundle/CatalogBundle/Model/AbstractProduct.php [hunk 2]
```diff
@@ -554,29 +554,65 @@ public function setEnabled($enabled)
...
-        if (null !== $this->family && $this->family->getAttributes()->contains($attribute)) {
-            return false;
-        }
+        return null !== $this->family && $this->family->getAttributes()->contains($attribute);
+    }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function hasAttributeInGroups(AttributeInterface $attribute)
```
Comment (jjanvier) [inline src/Pim/Bundle/CatalogBundle/Model/AbstractProduct.php]: hasAttributeInVariantGroup

**Comment by jjanvier** [comment]
GTM except tiny comments

**Comment by solivier** [comment]
GTM

**Pull request merged**